pytest_fixtures/component/maintain.py

```diff
@@ -29,34 +29,38 @@ def _finalize():
         assert sat_maintain.execute('rm -rf /tmp/backup-*').status == 0
 
 
-@pytest.fixture(scope='module')
-def module_synced_repos(sat_maintain, session_capsule_configured, module_sca_manifest):
-    org = sat_maintain.api.Organization().create()
-    sat_maintain.upload_manifest(org.id, module_sca_manifest.content)
+@pytest.fixture(scope='session')
+def session_synced_repos(
+    sat_maintain, session_target_sat, session_capsule_configured, session_sca_manifest
+):
+    org = session_target_sat.api.Organization().create()
+    session_target_sat.upload_manifest(org.id, session_sca_manifest.content)
     # sync custom repo
-    cust_prod = sat_maintain.api.Product(organization=org).create()
-    cust_repo = sat_maintain.api.Repository(
+    cust_prod = session_target_sat.api.Product(organization=org).create()
```

Comment:
Do not replace `sat_maintain`
Instead ensure that `api` works always against `satellite`:

`sat_maintain.satellite.api` 

For Capsule it runs against its Satellite
For Satellite it runs against satellite itself